core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -105,7 +105,11 @@ contract PricelessPositionManager is FeePayer {
         withdrawalLiveness = _withdrawalLiveness;
         TokenFactory tf = TokenFactory(_tokenFactoryAddress);
         tokenCurrency = tf.createToken(_syntheticName, _syntheticSymbol, 18);
-        priceIdentifer = _priceFeedIdentifier;
+
+        IdentifierWhitelistInterface identiferWhiteList = _getIdentifierWhitelist();
+        require(identiferWhiteList.isIdentifierSupported(_priceIdentifier));
```

Comment:
nit: could probably simplify this to one line:
```
require(_getIdentifierWhitelist().isIdentifierSupported(_priceIdentifier));
```

Reply:
sure & done